microsoft365/outlook-mcp/app/global_config.py

```diff
@@ -0,0 +1,19 @@
+"""Global configuration constants and variables for the Outlook Mail tool."""
```

Comment:
Is there any benefit to keeping these separate? We used to have different sets of tools in gptscript that were read only, but now I don't think this gives us any benefit

Reply:
fair. added an update to merge the scopes